### PR TITLE
Added aria-live='polite'

### PR DESCRIPTION
**Pull request by Hrittik20**
Issue: #5617

### CODE REVIEW

**Comment by stevepiercy** [comment]
@Hrittik20 CI checks need to pass. Same for backported PR #5640.

See [Change log entry](https://6.docs.plone.org/contributing/index.html#change-log-entry) and [Code quality](https://6.docs.plone.org/volto/contributing/index.html#code-quality)

**Comment by Hrittik20** [comment]
@stevepiercy Sorry, I forgot to add change log entry

File: packages/volto/news/5617.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+Added aria-live='polite' in `Contents.jsx` to improve accessibility. @Hrittik20
```
Comment (stevepiercy) [inline packages/volto/news/5617.feature:1]:
```suggestion
Added `aria-live="polite"` in `Contents.jsx` to improve accessibility for the Contents page. @Hrittik20
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Review by stevepiercy — COMMENTED** [review]
This is good. Please check ESLint.

**Comment by stevepiercy** [comment]
@Hrittik20 do you get notifications from GitHub of failed GitHub workflows? I get them whenever I submit a pull request that breaks them, so I assume you would too. You can click each failed workflow's "Details" link to see what might cause the failure.

Anyway, there are two workflows that you need to address. I would suggest that you run tests locally to save time, effort, and frustration. See [Testing](https://6.docs.plone.org/volto/contributing/testing.html) and [Acceptance testing](https://6.docs.plone.org/volto/contributing/acceptance-tests.html).

**Comment by Hrittik20** [comment]
@stevepiercy Yes, I am working on it. I am getting few errors while linting and testing so trying to fix it now

**Review by stevepiercy — APPROVED** [review]
I approve the change log. A core maintainer needs to review the rest.

**Review by giuliaghisini — APPROVED** [review]

**Review by davisagli — APPROVED** [review]

**Comment by giuliaghisini** [comment]
@davisagli @stevepiercy @Hrittik20 is it possible to have this also in Volto 17?

**Comment by stevepiercy** [comment]
@giuliaghisini you mean, like in #5640? :wink:

**Comment by giuliaghisini** [comment]
> @giuliaghisini you mean, like in #5640? 😉

yes!

**Pull request merged**